source/whats-new.txt

```diff
@@ -58,6 +58,8 @@ The 5.0 driver release introduces the following features:
   ``Decimal128`` values. In particular, the error responses for the
   ``isNumber()`` and ``asNumber()`` methods match the Java responses for
   equivalent ``Decimal128`` values.
+  
+- This driver deprecates the ``org.mongodb.scala.Observable.completeWithUnit`` method
```

Comment:
S: For consistency with other bullets and to avoid repeating "this driver," which is already in the intro

```suggestion
- The ``org.mongodb.scala.Observable.completeWithUnit()`` method is deprecated.
```